extensions/amp-form/0.1/amp-form.js

```diff
@@ -458,14 +495,8 @@ export class AmpForm {
     let p;
     if (this.ssrTemplateHelper_.isSupported()) {
       p = varSubPromise.then(() => {
-        this.actions_.trigger(
-            this.form_, FormEvents.SUBMIT, /* event */ null, trust);
-        // Note that we do not render templates for the submitting state
-        // and only deal with submit-success or submit-error.
-        return this.ssrTemplateHelper_.fetchAndRenderTemplate(this.form_);
-      }).then(
-          resp => this.handleSsrTemplateSuccess_(resp),
-          error => this.handleSsrTemplateFailure_(/** @type {!JsonObject} */ (error)));
+        this.handleSsrTemplate_(trust);
```

Comment:
We need to return the promise to chain it, right? Note that only short-form syntax automatically returns:

```js
(() => 1)(); // Returns 1.
(() => { 1; })(); // Does not return 1.
```